web/src/main/java/org/openmrs/web/WebUtil.java

```diff
@@ -420,23 +418,19 @@ public boolean supportsPropertyName(String propertyName) {
 		return OpenmrsConstants.GP_SEARCH_DATE_DISPLAY_FORMAT.equals(propertyName);
 	}
 	
-	public static void setDefaultDateCache(String defaultDateCache) {
-		WebUtil.defaultDateCache = defaultDateCache;
-	}
-	
 	/**
 	 * @see org.openmrs.api.GlobalPropertyListener#globalPropertyChanged(org.openmrs.GlobalProperty)
 	 */
 	@Override
 	public void globalPropertyChanged(GlobalProperty newValue) {
-		setDefaultDateCache(null);
+		throw new UnsupportedOperationException();
 	}
 	
 	/**
 	 * @see org.openmrs.api.GlobalPropertyListener#globalPropertyDeleted(java.lang.String)
 	 */
 	@Override
 	public void globalPropertyDeleted(String propertyName) {
-		setDefaultDateCache(null);
+		throw new UnsupportedOperationException();
```

Comment:
Why throw an exception?

Reply:
Well since the property `defaultDateCache` was removed prior to this issue requirements, this method was just initially nullifying that property. So this Operation is no longer supported.